src/reducers/calculator.js

```diff
@@ -1,7 +1,7 @@
 import * as actionTypes from "../utils/actionTypes";
 import * as operators from "../utils/actionTypes"
 
-const initialState ={
+export const initialState ={
```

Comment:
as far as I see you do not use this outside of calculator, so it doesn't need to to be exported.

Reply:
I will remove it, I accidentally left it after try to do something.